Bio/SeqIO/PdbIO.py

```diff
@@ -11,12 +11,30 @@
 import warnings
 
 from Bio import BiopythonParserWarning
-from Bio.Data.SCOPData import protein_letters_3to1
+from Bio.Data.SCOPData import protein_letters_3to1 as scop_3to1
+from Bio.Data.IUPACData import protein_letters_3to1_extended as iupac_3to1_ext
 from Bio.Seq import Seq
 from Bio.SeqRecord import SeqRecord
 from .Interfaces import SequenceIterator
 
 
+_aa3to1_dict = {}
+_aa3to1_dict.update(iupac_3to1_ext)
+_aa3to1_dict.update(scop_3to1)
+
+
+def _res2aacode(residue, undef_code="X"):
+    """Return the one-letter amino acid code from the residue name.
+
+    Non-amino acid are returned as "X".
+    """
+    return (
```

Comment:
I would prefer this being more explicit, but it's OK like this as well!

Reply:
I agree, this version is five lines - an if/else would be four lines, and clearer.